deployments/roman/environments/post-start-hook

```diff
@@ -38,7 +38,9 @@ set +e   # Do not let init failures block logins
 /opt/common-scripts/git-sync-v4 https://github.com/spacetelescope/roman_notebooks \
                                 main \
                                 /home/jovyan/references/roman_notebooks
+
+conda run -n roman-cal pip install jdaviz
```

Comment:
While this may work, I'm unsure if this is a good idea. It may considerably increase the boot time and prevent user logins if something goes wrong. If there is no other alternative, perhaps we can add `|| true` at the end to continue the boot process if there are any failures.